src/platform/common/process/currentProcess.node.ts

```diff
@@ -5,6 +5,9 @@
 import { injectable } from 'inversify';
 import { EnvironmentVariables } from '../variables/types';
 
+/**
+ * Used to wrap process.env. Potentially overkill?
+ */
```

Comment:
It's only used in `environmentActivationService.node.ts`, we can probably inline it.